src/snapred/backend/data/LocalDataService.py

```diff
@@ -501,6 +501,10 @@ def _getLatestReductionVersionNumber(self, runNumber: str, useLiteMode: bool) ->
 
     ##### NORMALIZATION METHODS #####
 
+    def normalizationExists(self, runId: str, useLiteMode: bool) -> bool:
+        version = self._getVersionFromNormalizationIndex(runId, useLiteMode)
+        return version is not None
```

Comment:
This will become
``` python
def normalziationExists(self, runId: str, useLiteMode: bool) -> bool:
    version = self.normalizationIndexor(runId, useLiteMode).currentVersion()
    return version is not UNINITIALIZED
```

Reply:
Or if you want to get the latest applicable version, replace `currentVersion()` with `latestApplicableVersion(runId)`